arch/cpu/arm/cortex-m/trustzone/normal/Makefile.normal

```diff
@@ -0,0 +1,8 @@
+# Exclude the normal dbg-arch.c from the build
+CONTIKI_SOURCES_EXCLUDES += dbg-arch.c
+
+ifndef TRUSTZONE_SECURE_WORLD_PATH
+  $(error TRUSTZONE_SECURE_WORLD_PATH must be set.)
+endif
+
+LDFLAGS += $(TRUSTZONE_SECURE_WORLD_PATH)/$(BUILD_DIR_BOARD)/$(CMSE_IMPORT_LIB)
```

Comment:
If you put the last five lines under the `else ifeq ($(TRUSTZONE_SECURE_BUILD), 0)` block in Makefile.cortex-m instead, this Makefile can be removed completely.